File: conditionals/timeofday.c
// Cairo Taylor, Time of Day
#include <stdio.h>
#include <time.h>

int main(void){
    time_t seconds;
    seconds = time(NULL);
    time_t rawtime;
    struct tm * timeinfo;
    time(&rawtime);
    timeinfo = localtime(&rawtime);
    time_t now = time(NULL);
    struct tm * tm_struct = localtime(&now);
    int hour = tm_struct->tm_hour;

if(hour == 12){
    printf("It's noontime!");
}else if(hour >= 2 && hour < 6){
    printf("Dawn is upon you! You sure do get up early!");
}else if(hour >= 6 && hour < 12){
    printf("Good Morning!");
}else if(hour >= 12 && hour < 20){
    printf("Good evening!");
}else if(hour >= 20 && hour < 22){
    printf("It's getting late!");
}else{
    printf("Go to sleep, it's nighttime.");
}
    return 0;
}

File: conditionals/time of day.py
# Cairo Taylor, Time of Day

import time
current = time.time()
now = time.ctime(current)
local_time = time.localtime(current)
minutes = local_time.tm_min
hours = local_time.tm_hour

if hours == 12:
    print("It's noontime!")
elif hours >= 2 and hours < 6:
    print("Dawn is upon you! You sure do get up early!")
elif hours >= 6 and hours < 12:
    print("Good Morning!")
elif hours >= 12 and hours < 20:
    print("Good evening!")
elif hours >= 20 and hours < 22:
    print("It's getting late!")
else:
    print("Go to sleep, it's nighttime.")